resourcepool/resourcepool.go

```diff
@@ -0,0 +1,9 @@
+package resourcepool
+
+const (
```

Comment:
Can you also add in more constants so we can get rid of other constants.go spread across:

```
	// Resource Pools
	Elastic    = "elastic"
	ElasticGPU = "elasticGpu"
	Reserved   = "reserved"
```